pkg/osquery/extension.go

```diff
@@ -62,6 +62,9 @@ const (
 	// Bucket name to use for buffered result logs.
 	resultLogsBucket = "result_logs"
 
+	// the bucket which we push values into from server-backed tables, like kolide_target_membership
+	ServerProvidedDataBucket = "server_provided_data"
```

Comment:
I don't think this symbol needs to be exported.

Reply:
this does, so that we can refer to the bucket from the table package - i'll actually have to do the same with the launcher identifier bucket when i move that table out into the 'platform tables' section